ch4/exercise4-10/issues.go

```diff
@@ -4,19 +4,59 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"time"
 
 	"./github"
 )
 
-//!+
+const (
+	Day = time.Hour * 24
+
+	Year = Day * 365
+
+	Month = Year / 12
```

Comment:
I didn't realize this was necessarily wrong. I did this exercise with Forrest so I thought it was ok, but good to know it wouldn't fly in production code.

Reply:
Relative intervals do have a place, but they have different language around them, and year/12 doesn't have a place. For instance, in a UI offering both (see metrics like Grafana), I'd expect to see:

* "last 30 days" -> now - 30 * day
* "last 24 hours" -> now - 30 * hour
* "last 365 days" -> now - 365 * day

As opposed to:

* "today" -> since 00:00 on this day
* "yesterday -> 00:00 yesterday to 00:00 today
* "this month" -> since 00:00 on the 1st
* "this year" -> since 00:00 on January 1st
* "last year" -> 00:00 Jan 1 to 00:00 Jan 1 for the previous year.

Note this is pretty close to `last 1 year` so many dashboards will use "last 365 days" instead of a year -- also because 1 year != 365 days (leap years, leap seconds) so it is more complex math to calculate the proper last 1 year.